src/lib/plugins/cpu.js

```diff
@@ -98,33 +98,36 @@ export function cpuFactory( React, colors ) {
       return averageCpuUsage
     }
 
-    getColor( cpuAverage ) {
-      const CPU_USAGE_COLORS = {
-        HIGH: colors.lightRed,
-        MODERATE: colors.lightYellow,
-        LOW: colors.lightGreen
-      }
+    getColor(cpuAverage) {
+      const colors = this.props.options.colors
 
       if ( cpuAverage < 50 ) {
```

Comment:
Won't be possible, since switch statements compare equality of switched values to the evaluated version of the expression as part of the case statement. E.g. you can switch a variable and compare it directly to a particular value, but the use case here is different :)